pkg/config/config.go

```diff
@@ -151,7 +152,9 @@ func ParseConfigBytes(data []byte) error {
 			return errors.Errorf("missing ethereum wallet private key environment variable '%v'", PrivateKeyEnvName)
 		}
 	}
-
+	if len(config.MainnetRPC) ==0{
+		config.MainnetRPC = config.NodeURL;
```

Comment:
maybe to avoid maintaining 2 different configs can just add

config.NodeURLMainnet;
config.NodeURLTestnet;

Reply:
Now I see that many settings are different for mainnet vs testnet so this will probably bloat the settings file.